test_cases/arc4_types/address.py

```diff
@@ -0,0 +1,24 @@
+from algopy import Contract, Global, Txn, arc4
+
+SOME_ADDRESS = "VCMJKWOY5P5P7SKMZFFOCEROPJCZOTIJMNIYNUCKH7LRO45JMJP6UYBIJA"
+
+
+class Arc4AddressContract(Contract):
+    def approval_program(self) -> bool:
+        address = arc4.Address(Txn.sender)
+        assert address == Txn.sender
+        assert address.length == 32
+        assert address.native == Txn.sender
+
+        zero_address = arc4.Address(Global.zero_address.bytes)
```

Comment:
I think we can do `arc4.Address()` now right?

Reply:
(to get a zero address)

Reply:
Yeah, but in this case I wanted to cover the constructor case that takes a bytes, `Global.zero_address` was just a convenient source of another address

Reply:
ah ok, I just use `Txn.sender.bytes` :)